src/main/java/uk/gov/hmcts/reform/civil/model/dq/FixedRecoverableCosts.java

```diff
@@ -16,4 +17,7 @@ public class FixedRecoverableCosts {
     private ComplexityBand band;
     private YesOrNo complexityBandingAgreed;
     private String reasons;
+    // below used in intermediate claim FRC
+    private YesOrNo agreedWithOtherParty;
```

Comment:
Don't think this is needed. 
we can reuse this field `complexityBandingAgreed`

Reply:
done